java/code/src/com/redhat/rhn/frontend/xmlrpc/serializer/BootstrapSystemOverviewSerializer.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2018 SUSE LLC
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.rhn.frontend.xmlrpc.serializer;
+
+import com.redhat.rhn.frontend.dto.BootstrapSystemOverview;
+import com.redhat.rhn.frontend.xmlrpc.serializer.util.SerializerHelper;
+import com.redhat.rhn.frontend.xmlrpc.serializer.util.SystemSerializerUtils;
+import redstone.xmlrpc.XmlRpcException;
+import redstone.xmlrpc.XmlRpcSerializer;
+
+import java.io.IOException;
+import java.io.Writer;
+
+
+/**
+ *
+ * BootstrapSystemOverviewSerializer
+ *
+ * @xmlrpc.doc
+ *
+ * #struct("system")
+ *     #prop("int", "id")
+ *     #prop("string", "name")
+ *     #prop_desc("dateTime.iso8601",  "last_checkin", "Last time server
+ *             successfully checked in")
+ *     #prop_desc("dateTime.iso8601",  "created", "Server registration time")
+ *     #prop_desc("dateTime.iso8601",  "last_boot", "Last server boot time")
+ *     #prop_desc("int",  "extra_pkg_count", "Number of packages not belonging
+ *             to any assigned channel")
+ *     #prop_desc("int",  "outdated_pkg_count", "Number of out-of-date packages")
+ *     #array_single("string", "hw_addresses")
```

Comment:
Not sure if this works. `ErrataHandler.java` has the following example:

```
     *          #prop_desc("array", "keywords", "'keywords' is the key into the struct")
     *              #array_single("string", "keyword - List of keywords to associate with the errata.")
```
So maybe it needs to be part of a `prop_desc`

Reply:
These xmlrpc docs!

Reply:
Honestly, I don't have a clue about the syntax of it. I changed to something that could be right, but I need to check the results after.

Reply:
The best would be if you could build the package. Currently this is only processed during package build.
In the git root do:
```
$> TEST=1 rel-eng/build-packages-for-obs.sh spacewalk-java
=== Building package [spacewalk-java-2.8.78.12-1] from java/ (Try 1)
/tmp/push-packages-to-obs/SRPMS/spacewalk-java /cvs/GALAXY/Manager-3.2/java
```
Now the package sources (tar, spec and changes) are available in `/tmp/push-packages-to-obs/SRPMS/spacewalk-java` . You can copy it to an OSC checkout of Devel:Galaxy:Manager:Head/spacewalk-java and build it locally with `iosc build --no-verify SLE_12_SP3 x86_64` . If it works - ok - if it crash you have introduced a syntax error :-)